scripts/_GameMaker.js

```diff
@@ -2306,10 +2312,10 @@ function GameMaker_Tick()
 
 		        case ROOM_ENDOFGAME:
                 case ROOM_ABORTGAME:
-                    Run_EndGame();
+                    Run_EndGame(false);
                     return;

```

Comment:
Since you're touching this part of code, could you please align this case statement with the others?

Reply:
Done - I applied the changes to the whole switch statement.